docs/tutorials/mobile/pose-detection.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Object detection and pose estimation with Yolov8
```

Comment:
nit: casing is YOLOv8 in the original repo so might be slightly better to follow that